projects/1.20/assets/Warriors of Past Epoch/warriorsofpastepoch/lang/zh_cn.json

```
@@ -0,0 +1,54 @@
+{
+  "effect.warriorsofpastepoch.disease_resistance": "疫疾抗性",
+  "effect.warriorsofpastepoch.contagious_touch_morbidity": "感染之触:染疫",
+  "effect.warriorsofpastepoch.second_hit": "",
+  "item.warriorsofpastepoch.musket": "§6穆什科特火枪",
+  "effect.warriorsofpastepoch.rough_wound": "极疮",
+  "effect.warriorsofpastepoch.plague_pestilence": "疫疾:瘟疫",
+  "item.warriorsofpastepoch.ammo_bag": "弹药袋",
+  "death.attack.black_death.item": "%1$s 被 %2$s 使用 %3$s 杀死了。",
```

```suggestion
  "death.attack.black_death.item": "%1$s被 %2$s使用%3$s杀死了。",
```


在1.16或更高版本下，英文字母、阿拉伯数字等非中文文本无需在前后添加空格。
具体可见我们的 [翻译规范](https://cfpa.site/TransRules/) 中 **04.排版** 一节下。
诸如这类的问题请在您翻译的所有文件中全部查明并修改。